nautobot/docs/user-guide/administration/installation/index.md

```diff
@@ -18,7 +21,79 @@ For more information about the Docker tags, Docker configurations, or using cont
 
     Nautobot *should* work on any POSIX-compliant system including practically any flavor of Linux, BSD, or even macOS, but those are not *officially* supported at this time.
 
-To begin installing Nautobot, click the link to your preferred deployment approach from the list above.
+
+#### Bare metal / Virtual Machine Pros
+
+- Familiar deployment option for many administrators
+- Performance is generally higher without the overhead of virtualization
+- Full control over hardware and software allows for greater customization and security practices
+
+#### Bare metal / Virtual Machine Cons
+
+- Maintenance and high availability is more difficult than Docker or Cloud options
+- Preparing server for production grade security and scaling requires additional effort 
+
+### Docker Compose
+
+Nautobot can be deployed via Docker using [Nautobot Docker images](../guides/docker.md) that are available either:
+
+* [Docker Hub](https://hub.docker.com/r/networktocode/nautobot)
```

Comment:
FYI - I think our markdownlint toolchain doesn't like mixing `- list` with `* list` style in the same file.

Reply:
ok I will swap out hyphens for stars to be consistent, good catch